lib/parser.rb

```diff
@@ -12,4 +15,14 @@ def print_first_line
   def close_file
     @file.close
   end
+
+  def number_of_lines
+    @lines = @file.count 
+  end
+
+  def format_to_object
+    number_of_lines()
+    object = { "#{@file_name}" => {"lines" => @lines }}
```

Comment:
```suggestion
    object = { "#{@file_name}" => {"lines" => number_of_lines }}
```

Seguindo minha sugestão anterior